torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -4841,6 +4841,9 @@ Tensor batch_norm_jvp(
     TORCH_INTERNAL_ASSERT(
         running_mean.has_value() && running_var.has_value(),
         "Expect running_mean and running_var to have value when train=false");
+    TORCH_CHECK(
+        !running_mean.value()._fw_grad(/*level=*/0).defined() && !running_mean.value()._fw_grad(/*level=*/0).defined(),
+        "batch_norm_jvp expects running_mean and running_var to not have forward grad defined");
```

Comment:
Nit: "running_mean and running_var are not differentiable, you attempted to differentiate them while performing forward-mode AD on batch_norm" would be clearer to the user?